src/provider.rs

```diff
@@ -98,7 +98,12 @@ impl SchemaProvider for SeafowlSchema {
             },
         };
 
-        delta_table.load().await?;
+        delta_table.load().await.map_err(|e| {
+            DataFusionError::Context(
+                format!("loading Delta Table {}.{}", self.name, name),
```

Comment:
Might be also useful to include the table url as well.

Reply:
good idea fixed

```
Loading Delta Table public.replicated_table (memory:///61b8bdab-45dd-479b-a7d5-d56dde4e56f7)
caused by
External error: Not a Delta table: no log files
```

Reply:
Also I did not realize we could use memory tables!

Reply:
Yep, you can create an in-memory storage location and write a table there, but it will go away on Seafowl restart :)